public/js/blog-card-filter-options.js

```diff
@@ -0,0 +1,23 @@
+function reverseString(str) {
+    return str.split("-").reverse().join("/");
+}
+
+$(document).ready(function(){
```

Comment:
יכולה להשתמש פה ב:
```suggestion
$( () => { 
...
})
```

Reply:
אלו שתי פעולות שעושות את אותו הדבר אין באמת עדיפות ללמבדה על פונקציה והפוך :)

Reply:
מסכים אבל אם אפשר "להתחנף" למרצה בכתיב מקוצר ולהראות אלגנטיות מה אכפת לנו?

Reply:
הajax לא אוהב את הלאמבדה(אחרי שינוי הוא לא יודע להשתמש בthis כמו בfunction ככה שזה מקריס את הסינון), גם לא מצאתי אנשים שמשתמשים בלמבדה בתוך ajax ברחבי הגלובוס

Reply:
בכל מקרה היה חלק בודד למעלה שהשינוי לא פגם ככה שבו החלפתי